### PR TITLE
Update VHD with new images

### PR DESCRIPTION
**Pull request by xuto2**

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -452,37 +464,40 @@ ls -ltr /usr/local/bin >> ${VHD_LOGS_FILEPATH}
...
+# v1.17.9-hotfix.20200714.1
+# v1.18.4-hotfix.20200624.1
+# v1.18.6-hotfix.20200714.1
 PATCHED_HYPERKUBE_IMAGES="
 1.14.7-hotfix.20200408.1
 1.14.8-hotfix.20200529.1
 1.15.7-hotfix.20200326
 1.15.10-hotfix.20200408.1
 1.15.11-hotfix.20200529.1
-1.15.11-hotfix.20200529.2
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:468]: .2 are from Liming and the VHD is never released so overriding it

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -452,37 +464,40 @@ ls -ltr /usr/local/bin >> ${VHD_LOGS_FILEPATH}
...
+1.15.11-hotfix.20200714.1
 1.15.12-hotfix.20200623.1
-1.15.12-hotfix.20200623.2
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:470]: .2 are from Liming and the VHD is never released so overriding it

File: vhdbuilder/packer/install-dependencies.sh [hunk 3]
```diff
@@ -452,37 +464,40 @@ ls -ltr /usr/local/bin >> ${VHD_LOGS_FILEPATH}
...
+1.15.12-hotfix.20200714.1
 1.16.7-hotfix.20200601.1
 1.16.8.1
 1.16.9-hotfix.20200529.1
 1.16.10-hotfix.20200623.1
-1.16.10-hotfix.20200623.2
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:475]: .2 are from Liming and the VHD is never released so overriding it

File: vhdbuilder/packer/install-dependencies.sh [hunk 4]
```diff
@@ -452,37 +464,40 @@ ls -ltr /usr/local/bin >> ${VHD_LOGS_FILEPATH}
...
 1.17.3-hotfix.20200601.1
 1.17.4.1
 1.17.5.1
 1.17.7-hotfix.20200624
-1.17.7-hotfix.20200624.1
+1.17.7-hotfix.20200714.1
+1.17.9-hotfix.20200714.1
 1.18.1.1
 1.18.2-hotfix.20200624
-1.18.2-hotfix.20200624.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:483]: from Liming and the VHD is never released so overriding it

File: vhdbuilder/packer/install-dependencies.sh [hunk 5]
```diff
@@ -452,37 +464,40 @@ ls -ltr /usr/local/bin >> ${VHD_LOGS_FILEPATH}
...
 1.16.9-hotfix.20200529.1
 1.16.10-hotfix.20200623.1
-1.16.10-hotfix.20200623.2
+1.16.10-hotfix.20200714.1
+1.16.13-hotfix.20200714.1
 1.17.3-hotfix.20200601.1
 1.17.4.1
 1.17.5.1
 1.17.7-hotfix.20200624
-1.17.7-hotfix.20200624.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:480]: from Liming and the VHD is never released so overriding it

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:480]: https://github.com/Azure/AgentBaker/pull/155/files

File: vhdbuilder/packer/install-dependencies.sh (hunk 4 above)
Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:483]: https://github.com/Azure/AgentBaker/pull/155/files

**Comment by bowang-666** [comment]
seems build is failing at:
Executed "docker pull mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.6-hotfix.20200714.1" 60 times

**Pull request merged**